front/src/modules/ui/content-display/components/NumberDisplay.tsx

```diff
@@ -0,0 +1,20 @@
+import styled from '@emotion/styled';
+
+const StyledNumberDisplay = styled.div`
+  overflow: hidden;
+  text-overflow: ellipsis;
+  white-space: nowrap;
+  width: 100%;
+`;
+
+type OwnProps = {
+  value: string;
+};
+
+export function NumberDisplay({ value }: OwnProps) {
+  return (
+    <StyledNumberDisplay>
+      {value && Number(value).toLocaleString()}
```

Comment:
@brendanlaschke could you extract this to a utils/format/number.ts file and add tests on it to test a few different cases?